### ISSUE TITLE
Interpreting the YOLO output

### ISSUE THREAD

**Issue by oo92** [AlexeyAB/darknet#7368]
I am currently training on a dataset and I want to make sense of the output from YOLOv4-tiny.

`v3 (giou loss, Normalizer: (iou: 0.50, obj: 1.00, cls: 1.00) Region 30 Avg (IOU: 0.000000), count: 1, class_loss = 0.000007, iou_loss = 0.000000, total_loss = 0.000007 
v3 (giou loss, Normalizer: (iou: 0.50, obj: 1.00, cls: 1.00) Region 37 Avg (IOU: 0.773155), count: 5, class_loss = 1.460724, iou_loss = 3136.319092, total_loss = 3137.779785 
 total_bbox = 38508, rewritten_bbox = 0.000000 % 
v3 (giou loss, Normalizer: (iou: 0.50, obj: 1.00, cls: 1.00) Region 30 Avg (IOU: 0.000000), count: 1, class_loss = 0.000012, iou_loss = 0.000000, total_loss = 0.000012 
v3 (giou loss, Normalizer: (iou: 0.50, obj: 1.00, cls: 1.00) Region 37 Avg (IOU: 0.800353), count: 3, class_loss = 1.261792, iou_loss = 1423.418457, total_loss = 1424.680298 
 total_bbox = 38511, rewritten_bbox = 0.000000 % `

1. What is iou:0.50? It seems to me like a threshold of some kind? Why is it there? As well as obj and cis?

2. What is count? Is it counting the classes in an image?

3. Why is my top IOU 0.00000 but the bottom IOU is 0.773? Why is the top just 0s but the bottom one is higher? Its started to do this only when I began using a new dataset.

4. What is Region 30, 37?

5. What would make rewritten_bbox 0.00000%? Why is total bbox so high when I only have, at most, 3 objects in an image?

**Comment by asfarley** [comment]
iou is intersection over union, it's a bounding-box error metric. Not sure about the rest.
